pkg/transformer/kubernetes/kubernetes.go

```diff
@@ -797,6 +797,21 @@ func (k *Kubernetes) Deploy(komposeObject kobject.KomposeObject, opt kobject.Con
 				return err
 			}
 			log.Infof("Successfully created Deployment: %s", t.Name)
+
+		case *extensions.DaemonSet:
```

Comment:
Not too sure why this code is being added? Can you describe this at least within the commit description?

Reply:
Sure I will mention

Reply:
@cdrage , updated commit message

Reply:
please update GitHub description too...